utils/utils.go

```diff
@@ -0,0 +1,19 @@
+package utils
```

Comment:
What do you think about naming it `error` or something similar? If we have only error handling here, I prefer to be more specific with package names.

Reply:
Yes - the `utils` package is an anti-pattern, we, Go developers should never create. 

It's like a mantra =D 

NO `UTILS`. Maybe `common` (🙃 sometimes when drunk). 

Otherwise, let's aim always for some single ~functionality purpose package like `httperr` in this case. Otherwise,  it can grow into some nice meaningless garbage package.